File: src/main/java/dev/expx/dependencymanager/resolver/impl/SimpleLibraryStore.java
package dev.expx.dependencymanager.resolver.impl;


import dev.expx.dependencymanager.resolver.lib.LibraryStore;

import java.nio.file.Path;
import java.util.ArrayList;
import java.util.List;

/**
 * @author PaperMC
 */
public class SimpleLibraryStore implements LibraryStore {
    private final List<Path> paths = new ArrayList();

    public SimpleLibraryStore() {
    }

    public void addLibrary(Path library) {
        this.paths.add(library);
    }

    public List<Path> getPaths() {
        return this.paths;
    }
}

File: src/main/kotlin/dev/expx/dependencies/resolver/impl/SimpleLibraryStore.kt
package dev.expx.dependencies.resolver.impl

import dev.expx.dependencies.resolver.lib.LibraryStore
import java.nio.file.Path

/**
 * @author PaperMC
 */
class SimpleLibraryStore : LibraryStore {
    val paths: MutableList<Path?> = ArrayList()

    override fun addLibrary(var1: Path) {
        this.paths.add(var1)
    }
}